zh/deploy-tidb-enterprise-edition.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 在 Kubernetes 上部署 TiDB 企业版
+summary: 了解如何在 Kubernetes 上部署 TiDB 企业版。
+---
+
+# 部署 TiDB 企业版
+
+本文档介绍如何在 Kubernetes 上部署 TiDB 集群企业版及相应的企业版工具。
```

Comment:
Can we add a few words to explain what is the enterprise version?

Reply:
https://pingcap.com/download-cn/
Use the description from here.